mne/decoding/search_light.py

```diff
@@ -638,3 +641,18 @@ def _gl_score(estimators, scoring, X, y):
                 score = np.zeros(shape, dtype)
             score[ii, jj, ...] = _score
     return score
+
+
+def _fix_auc(scoring, y):
+    from sklearn.preprocessing import LabelEncoder
+    # This fixes sklearn's inability to compute roc_auc when y not in [0, 1]
+    # scikit-learn/scikit-learn#6874
+    if scoring is not None:
+        if hasattr(scoring, '_score_func'):
+            if hasattr(scoring._score_func, '__name__'):
+                if scoring._score_func.__name__ == 'roc_auc_score':
```

Comment:
I had done this with '&' and it didn't work, but hadn't thought it'd differ from 'and'. Learn something everyday ..